docs/src/pages.ts

```diff
@@ -188,18 +188,7 @@ const pages: readonly MuiPage[] = [
         subheader: '/components/lab',
         children: [
           { pathname: '/components/about-the-lab', title: 'About the lab 🧪' },
-          {
-            pathname: '/components',
-            subheader: '/components/lab-pickers',
-            title: 'Date / Time',
-            children: [
-              { pathname: '/components/pickers', title: 'Introduction' },
-              { pathname: '/components/date-picker' },
-              { pathname: '/components/date-range-picker', title: 'Date Range Picker ⚡️' },
-              { pathname: '/components/date-time-picker' },
-              { pathname: '/components/time-picker' },
-            ],
-          },
+          { pathname: '/components/lab-date-and-time-pickers', title: 'Date & Time Pickers' },
```

Comment:
I was expecting this page to exists on the current docs too, but as we are going to release everything at once, it should be fine to exists only on `material-ui/lab-date-and-time-pickers`. Having this said, should we just remove this entry here? cc @siriwatknp does this changes anything for you?

Reply:
That's fine. It will be removed anyway. It is safe to be merged.